<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeslotsManager</class>
 <widget class="QDialog" name="TimeslotsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horaires de kholles</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listKholleurs">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_jour">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Jour :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Lundi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mardi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mercredi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jeudi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vendredi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Samedi</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_debut">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>DÃ©but de la kholle :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listTimeslots">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copier...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyAllButton">
       <property name="text">
        <string>Tout copier...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>TimeslotsManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
